README.md

```diff
@@ -47,16 +47,21 @@ plasma_cli get utxos --watcher=<watcher URL> --address=<public_address>
 plasma_cli get balance --watcher=<watcher URL> --address=<public_address>
 ```
 
-## Send ETH in the OMG Network
+## Sending ETH UTXO
 
-NOTE: This function will either
+This function will either
```

Comment:
Perhaps it's best to include an additional flag here so the user knows exactly which behaviour will occur?

Reply:
so, keep the `NOTE:` flag on ?

Reply:
Perhaps some documentation explaining why UTXO splits -> send to self are used? People used to developing in the Ethereum world may not understand why this is required (thinking of devs new to Plasma). If you don't think this is the right place to describe this behaviour maybe we could add a link to some knowledge base?

Reply:
yeah, devs new to Plasma will have a tough time grasping UTXO from the Readme alone.. I will have a dedicated supporting document for it 👌